libraries/botbuilder/package.json

```diff
@@ -20,9 +20,11 @@
   "main": "./lib/index.js",
   "typings": "./lib/index.d.ts",
   "dependencies": {
+    "@azure/ms-rest-js": "1.2.6",
     "@types/node": "^10.12.18",
     "botbuilder-core": "4.1.6",
     "botframework-connector": "4.1.6",
+    "botframework-streaming": "4.1.6",
```

Comment:
I just stayed consistent with the rest of packages for now. What version should we ref? ^4.5?

Reply:
It will work with 4.1.6, we just don't technically support it. I think our options are:
1 - Update the rest of the packages to the current release versions.
2 - Go with 4.1.6 for consistency.
3 - Go with ^4.5 with the knowledge it creates a bad experience for developers trying to build off of master.